doc/nrf/libraries/bootloader/bl_storage.rst

```diff
@@ -7,45 +7,23 @@ Bootloader storage
    :local:
    :depth: 2
 
-The bootloader storage library is used by the :ref:`bootloader` to read from and write to a protected data area that contains the provisioned bootloader data.
-The library can also be included into the application to allow reading the provisioned data.
+The bootloader storage library is used to read and write one-time programmable (OTP) data and a non-volatile counter.
+It may be used by both the |NSIB| (NSIB) and the application, but the application has read access only.
 
-The provisioned data is stored in the one-time programmable (OTP) region of UICR, if available (for example, on nRF9160 and the nRF5340 application core).
-In this case, it can only be read from applications that are in the secure domain.
-If OTP is not available, the data is stored in a regular flash page, and the bootloader blocks write access to it before booting the next image.
-In this case, some functions do not work when they are called from the application.
+The library has functions for either reading or writing, or in some cases both reading and writing, the following:
 
-The library stores the following data:
-
-* Image slot sizes
+* Image slot addresses
+* A monotonic counter used to enforce anti-rollback protection
 * Hashes of public keys
 * Invalidation tokens used to revoke public keys
-* :ref:`Application versions <store_app_version>`
+* And other public key metadata
 
+The OTP region of the UICR is used when present on the nRF SoC (e.g. nRF9160 and nRF5340), otherwise the internal flash memory is used.
```

Comment:
```suggestion
The OTP region of the UICR is used when present on the nRF SoC (for example, nRF9160 and nRF5340), otherwise the internal flash memory is used.
```